app/models/lookup_keys/puppetclass_lookup_key.rb

```diff
@@ -40,4 +41,8 @@ def validate_default_value
   def puppet?
     true
   end
+
+  def check_override_selected
+    errors.add(:override, _("must be true to edit the parameter")) unless self.override || (self.changes.size == 1 && self.changes["description"].present?) || self.new_record?
```

Comment:
woah, unless must not contain more than one conditional. I second @dLobatog's comment about changing the unless to if, but for readability reasons (Daniel already made the relevant comments regarding the conditional itself)